environment/src/main/java/com/sequenceiq/environment/credential/v1/converter/gcp/GcpCredentialV1ParametersToGcpCredentialAttributesConverter.java

```diff
@@ -53,6 +67,8 @@ private P12Parameters getP12(P12Attributes p12) {
     private JsonParameters getJson(JsonAttributes json) {
         JsonParameters response = new JsonParameters();
         response.setCredentialJson(json.getCredentialJson());
+        // Old credential has no project id in the database
```

Comment:
maybe a condition would be safer than a comment

Reply:
this value will be null. I just wanted to mention it maybe in the future if we will have such question then it is easier to understand why we dont populate the value.

Reply:
But we are populating it even if it is a null value, that's why I would use a condition instead of the comment it would self-describe that it could be null and that field is only set when the value is not null.